app/views/dashboard/index.html.erb

```diff
@@ -86,7 +86,7 @@
     %>
   </div>
 
-  <div class="govuk-grid-row dashboard-row">
+  <div class="govuk-grid-row card-group">
```

Comment:
If the `dashboard-row` class isn't used any more, it should be safe to remove whatever CSS rules were specified for it in our CSS files.

It's always better to clean things up as we go, otherwise the CSS can become unwieldy with rules for unused classes. 😱 

Before doing so: I'd suggest making sure none of the other views use the `dashboard-row` class for anything.

Reply:
@ollietreend dashboard-row is used in one place sadly. On the contact_us page!

Reply:
@Katy600 I've just had a look at the Contact Us page and think it's safe to remove. It adjusts the styling very slightly – but I don't think `dashboard-row` was being particularly helpful on that page anyway.

As part of this PR, could you please remove the use of `dashboard-row` on the Contact Us page (no need to replace it with anything else), and remove the associated CSS styles from `application.scss`?

Thanks! 🙏🏻